backend/configs/db.js

```diff
@@ -1,9 +1,25 @@
 import mongoose from 'mongoose';
```

Comment:
if you need review of be part, please add be reviewer, I will check only fe part